strapi/strapi
# feat(Blocks): Move blocks

### What does it do?

Adds the ability to move the blocks around in editor

### How to test it?

On hover user should be able to see drag option and able to drag block and list-items.

Note: There are some intermittent errors coming from slate and react-dnd, like "Expected targetIds to be registered" will be handled in coming PR. Workaround for now: you can try to drag item slowly to make it work.
